geode-core/src/test/java/org/apache/geode/internal/cache/PartitionedRegionTest.java

```diff
@@ -249,28 +232,6 @@ public void createClearPRMessagesShouldCreateMessagePerBucket() {
     assertThat(msgs.size()).isEqualTo(3);
   }
 
-  @Test
-  public void sendEachMessagePerBucket() {
```

Comment:
Is the deleted test case covered else where? Or it no longer valid?

Reply:
This is testing the old way of messaging; where message is sent per bucket. That is not true anymore.